README.md

```diff
@@ -43,6 +50,31 @@ This Event source plugin from Dynatrace captures all problems from your Dynatrac
           name: dt-update-comments.yml
 ```
 
+## dt_webhook 
+
+The dt_webhook event-source plugin is capable of receiving events from Dynatrace via the "Send event to Event-Driven Ansible" workflow action of the Red Hat Ansible for Workflows integration.
```

Comment:
Here a link to the Dynatrace docu would be helpful.

Reply:
I will add the link to the Dynatrace Docs of the Red Hat Ansible for Workflows integration